integration-tests/basic-test-app/android/app/src/main/embrace-config.json

```diff
@@ -10,6 +10,10 @@
     },
     "view_config": {
       "enable_automatic_activity_capture": false
+    },
+    "networking": {
```

Comment:
would be good to add to this script as well once the iOS side is figured out: https://github.com/embrace-io/embrace-react-native-sdk/blob/97a2926b0f853d1ac418537287f2b4d3df144db7/integration-tests/set-embrace-config.js#L4

Reply:
covered!